uniffi_bindgen/src/bindings/swift/gen_swift.rs

```diff
@@ -220,23 +222,46 @@ mod filters {
     /// Lower a Swift type into an FFI type.
     ///
     /// This is used to pass arguments over the FFI, from Swift to Rust.
-    pub fn lower_swift(name: &dyn fmt::Display, _type_: &Type) -> Result<String, askama::Error> {
-        Ok(format!("{}.lower()", var_name_swift(name)?))
+    pub fn lower_swift(name: &dyn fmt::Display, type_: &Type) -> Result<String, askama::Error> {
+        match type_ {
+            Type::Duration => Ok(format!(
+                "{}.lower{}()",
+                var_name_swift(name)?,
+                type_swift(type_)?
+            )),
+            _ => Ok(format!("{}.lower()", var_name_swift(name)?)),
+        }
     }
 
     /// Lift a Swift type from an FFI type.
     ///
     /// This is used to receive values over the FFI, from Rust to Swift.
     pub fn lift_swift(name: &dyn fmt::Display, type_: &Type) -> Result<String, askama::Error> {
-        Ok(format!("{}.lift({})", type_swift(type_)?, name))
+        match type_ {
+            Type::Duration => Ok(format!(
+                "{}.lift{}({})",
+                type_swift(type_)?,
+                type_swift(type_)?,
```

Comment:
Similarly, please make this something like:

```
format!(
    "{}.lift{}({})",
    type_swift(type_)?,
    type_.canonical_type_name(),
    name,
)
```

The stylistic difference is more obvious in this case - you want one reference to the type as an actual type expression, and one to a string naming the type to be included as part of the method name.